astropy/coordinates/earth.py

```diff
@@ -395,6 +403,33 @@ def get_itrs(self, obstime=None):
                                      for the location of this object at the
                                      default ``obstime``.""")
 
+    def get_gcrs_posvel(self, obstime):
```

Comment:
Tiny, but can this be just `gcrs_posvel` -- I think the standard is to assume a calculation gets done (and your not "getting" the information from somewhere).